website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,162 @@
+
+2. **Compile button &mdash;** The Compile button compiles the SQL code from the active File Editor, irrespective of its save status, and outputs it to the Compiled Code tab.
+
+3. **Build button &mdash;** The build button allows users to quickly access dbt commands related to the active model in the File Editor. The available commands include dbt build, dbt test, and dbt run, with options to include only the current resource, the resource and its upstream dependencies, the resource and its downstream dependencies, or the resource with all dependencies. This menu is available for all executable nodes.
+
+3. **Format button &mdash;** The editor has a **Format** button that can reformat the contents of your files. For SQL files, it uses either `sqlfmt` or `sqlfluff`, and for Python files, it uses `black`.
+
+5. **Results tab &mdash;** The Results console tab displays the most recent Preview results in tabular format. 
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/results-console-tab.jpg" width="100%" title="Preview results show up in the Results console tab"/>
+
+6. **Compiled Code tab &mdash;** The Compile button triggers a compile invocation that generates compiled code, which is displayed in the Compiled Code tab.
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/compiled-code-console-tab.jpg" width="100%" title="Compile results show up in the Compiled Code tab"/>
+
+7. **Lineage tab &mdash;** The Lineage tab in the File Editor displays the active model's lineage or  <Term id="dag" />. By default, it shows two degrees of lineage in both directions (`2+model_name+2`), however you can change it to +model+ (full DAG).
+    - Double-click a node in the DAG to open that file in a new tab
+    - Expand the DAG and use node selection syntax (select or exclude) to view a subset of your DAG
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/lineage-console-tab.jpg" width="100%" title="View resource lineage in the Lineage tab"/>
+
+## Invocation history
+The Invocation History Drawer stores information on dbt invocations in the IDE. When you invoke a command (like execute a dbt command such as `dbt run`), the associated logs are displayed in the Invocation History Drawer. 
+
+You can open the drawer multiple ways: 
+- Clicking the `^` icon next to the Command bar on the lower left of the page
+- Typing a dbt command and pressing enter
+- Or pressing Control-backtick (or Ctrl + `)
```

Comment:
i think you need to escape the backtick for it to be rendered properly since backtick has a certain meaning in MD:

https://www.growingwiththeweb.com/2015/06/escaping-backtick-in-markdown.html 

```suggestion
- Or pressing Control-backtick (or Ctrl + \`)
```

Reply:
good tip- thank you 🙌